<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPhonebook</class>
 <widget class="QDialog" name="DlgPhonebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phonebook</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabView">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="leSearch">
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tblView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdd">
      <attribute name="title">
       <string>Add</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>271</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblPhone1">
            <property name="text">
             <string>Phone 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePhone1"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblPhone2">
            <property name="text">
             <string>Phone 2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblPhone3">
            <property name="text">
             <string>Phone 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lePhone2"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lePhone3"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pbClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>lePhone1</tabstop>
  <tabstop>lePhone2</tabstop>
  <tabstop>lePhone3</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>leSearch</tabstop>
  <tabstop>tblView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
